components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -45,24 +45,6 @@ public static ApplicationManagementService getInstance() {
         return ApplicationManagementServiceImpl.getInstance();
     }
 
-    /**
-     * Creates a service provider with basic information.First we need to create
-     * a role with the
-     * application name. Only the users in this role will be able to edit/update
-     * the application.The
-     * user will assigned to the created role.Internal roles used.
-     *
-     * @param serviceProvider Service Provider Name
-     * @param tenantDomain Tenant Domain
-     * @param username User Name
-     * @return
-     * @throws IdentityApplicationManagementException
-     * @deprecated  This method is replaced by {@link #createApplicationWithTemplate}
-     */
-    @Deprecated
```

Comment:
We are removing this API. 

But any client calling this API won't break since we have introduced a new API with the same signature that returns the created application.